packages/message-backend/src/create-email/templates/partials/paymentRow.hbs

```diff
@@ -0,0 +1,88 @@
+                                                                    </span>
+                                                </td>
+                                            </tr>
+
+                                            <tr>
+                                                <td width="80%" class="purchase_item">
+                                                                <span class="f-fallback">
+                                                                    {{I18n 'orderConfirmation.receipt.alv'}} ({{vatPercentage}}%)
+                                                                </span>
+                                                </td>
+                                                <td class="align-right" width="20%" class="purchase_item pb-1">
+                                                                <span class="f-fallback no-wrap">
+                                                                    {{priceVat}} €
+                                                                </span>
+                                                </td>
+                                            </tr>
+                                        {{/each}}
+                                        <tr>
+                                            <td width="80%" class="purchase_item" style="border-top: 1rem solid #ffffff">
+                                                                <span class="f-fallback">
+                                                                    {{I18n 'orderConfirmation.receipt.paidTotal'}}
+                                                                </span>
+                                            </td>
+                                            <td class="align-right" width="20%" class="purchase_item" style="border-top: 1rem solid #ffffff">
+                                                                <span class="f-fallback no-wrap">
+                                                                    {{order.payment.total}} €
+                                                                </span>
+                                            </td>
+                                        </tr>
+
+                                        <tr>
+                                            <td width="80%" class="purchase_item">
+                                                                <span class="f-fallback">
+                                                                    {{I18n 'orderConfirmation.receipt.paymentMethod'}}
+                                                                </span>
+                                            </td>
+                                            <td class="align-right" width="20%" class="purchase_item">
+                                                                <span class="f-fallback no-wrap">
+                                                                    {{order.payment.paymentMethod}}
```

Comment:
Should maybe use this to paymentMethodLabel from payment as paymentMethod is the code eg. "nordea". Same change is applied to kassa now.

Reply:
Changed to paymentMethodLabel